### PR TITLE
Zookeeper 3.6.3 ubi8 image

### PR DESCRIPTION
**Pull request by snehakpersistent**
Redhat ubi8 image for Zookeeper 3.6.3

### CODE REVIEW

**Comment by seth-priya** [comment]
@kandarpamalipeddi please review

File: z/zookeeper/Dockerfiles/3.6.3_ubi8/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Zookeeper Build steps
+```
+
+docker build -t zookeeper .
```
Comment (kandarpamalipeddi) [inline z/zookeeper/Dockerfiles/3.6.3_ubi8/README.md:4]: Either tag  it as `zookeeper-ubi8` 
or
run as `zookeeper:latest`

**Review by kandarpamalipeddi — CHANGES_REQUESTED** [review]

File: z/zookeeper/Dockerfiles/3.6.3_ubi8/README.md (hunk 1 above)
Comment (snehakpersistent) [reply z/zookeeper/Dockerfiles/3.6.3_ubi8/README.md:4]: Updated to zookeeper:latest

**Review by kandarpamalipeddi — APPROVED** [review]

**Comment by kandarpamalipeddi** [comment]
@seth-priya 
Changes are looks good to me.

**Pull request merged**